takwimu/profiles/census.py

```diff
@@ -1891,9 +1892,9 @@ def get_worldbank_data(geo, session, country, level):
             cereal_yield_kg_per_hectare, _ = get_stat_data(
                 ['cereal_yield_kg_per_hectare_year', ], geo, session, percent=False)
         except Exception as e:
-            print "\n\n\n\n\n\n\n\n\n\n"
-            print e.message
-            print "\n\n\n\n\n\n\n\n\n\n"
+            print("\n\n\n\n\n\n\n\n\n\n")
+            print(e.message)
```

Comment:
@phillipahereza the best way of getting error message from python exception is through [`str(e)`](https://stackoverflow.com/a/43453668). `message` will sometime not exist e.g. if you go to Zambia, it will fail. Should be fixed in a separate PR though.